detox/src/devices/Device.test.js

```diff
@@ -815,7 +815,7 @@ describe('Device', () => {
   async function launchAndTestBinaryPath(configuration) {
     const device = schemeDevice(configurationsMock.pathsTests, configuration);
 
-    await device.prepare();
+    await device.prepare({ checkBinaryPath: true });
```

Comment:
Seems to me like `checkBinaryPath` doesn't really belong in here,  is there any function call without `checkBinaryPath:true` ?
If this is to separate the web driver from iOS/Android drivers, then maybe this logic should reside in the drivers themselves.

Reply:
I'm not ecstatic about how I had to get the tests passing. If I remember correctly, I was fighting against code coverage, but perhaps the real solution is to add a separate test for `getAbsoluteBinaryPath`

Reply:
So does this mean you can get rid of checkBinaryPath: true?

Reply:
sorry I missed this reference. yes we can get rid of it